src/Vts/MonteCarlo/Controllers/HistoryDetectorController.cs

```diff
@@ -19,7 +18,6 @@ public class HistoryDetectorController : IDetectorController
         public HistoryDetectorController(IEnumerable<IHistoryDetector> detectors, ITissue tissue)
         {

```

Comment:
It appears that tissue doesn't do anything, I know we can't remove it but maybe we should keep a note of this so we can figure out what to do going forward.

Reply:
Added parameter description to state parameter "tissue" not used.